### PR TITLE
Inject OKTETO_TOKEN in okteto deploy

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
Signed-off-by: Agustin Ramiro Diaz <agustin.ramiro.diaz@gmail.com>

After this PR we'll have the 3 envvars injected into the deploy commands to be able to run things like
```
helm registry login ${OKTETO_REGISTRY_URL} -u ${OKTETO_USERNAME} -p ${OKTETO_TOKEN}
```

- `OKTETO_REGISTRY_URL` is already set at the beggining of the deploy command
- `OKTETO_USERNAME` is already set at the beggining of the context command, which gets run before the deploy command

# Proposed changes

- If not defined, add `OKTETO_TOKEN` from context as envvar in deploy command
- Mask `OKTETO_TOKEN`'s value

### CODE REVIEW

**Review by jLopezbarb — APPROVED** [review]

**Comment by AgustinRamiroDiaz** [comment]
@jLopezbarb Deepsource's analysis is stuck. Do you know what could be the issue?

File: pkg/model/const.go [hunk 1]
```diff
@@ -203,8 +203,8 @@ const (
 	// OktetoSkipCleanupEnvVar defines the okteto binary that should be used
 	OktetoSkipCleanupEnvVar = "OKTETO_SKIP_CLEANUP"
 
-	// OktetoUserEnvVar defines the user is using okteto
-	OktetoUserEnvVar = "OKTETO_USER"
+	// OktetoUserEnvVar defines the user using okteto
+	OktetoUserEnvVar = "OKTETO_USER_ID"
```
Comment (teresaromero) [inline pkg/model/const.go:207]: why are we changing this env var? `OKTETO_USER` is being used in different integration tests and defined at docs for the tests. Also is being added as env to the circleci jobs.

**Review by teresaromero — CHANGES_REQUESTED** [review]
check `OKTETO_USER` change is not breaking integration and ci pipelines

File: pkg/model/const.go (hunk 1 above)
Comment (AgustinRamiroDiaz) [reply pkg/model/const.go:207]: @teresaromero great catch! I don't know why my vscode didn't show me the references.

This comment also helped me see that there's a better envvar for this, `OKTETO_USERNAME`. I'll update the code and ping you again

**Review by AdrianPedriza — APPROVED** [review]

**Comment by ifbyol** [comment]
@AgustinRamiroDiaz Can you include the issue when creating the PR, please? It is easier to check what was the original issue.

**Review by ifbyol — APPROVED** [review]

**Comment by AgustinRamiroDiaz** [comment]
> @AgustinRamiroDiaz Can you include the issue when creating the PR, please? It is easier to check what was the original issue.

It appears under development in the side panel. Do you want me to add it in the description too?

**Comment by ifbyol** [comment]
> > @AgustinRamiroDiaz Can you include the issue when creating the PR, please? It is easier to check what was the original issue.
> 
> It appears under development in the side panel. Do you want me to add it in the description too?

I didn't see it, not sure if you added it now and it wasn't before or I didn't check it properly. Anyways, thanks!

**Review by teresaromero — APPROVED** [review]

**Pull request merged**